applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/AssessmentResults/Default.js

```diff
@@ -190,13 +190,35 @@ function flaggedFieldIsValid(flag, name) {
 
 function hasInvalidCustomFields() {        
     let invalidFieldsFound = false;
-    $("input[id^='custom']:visible").each(function (i, el) {
-        let fieldValidity = document.getElementById(el.id).validity.valid;        
-        if (!fieldValidity) {
-            invalidFieldsFound = true;
+    $("input[id^='custom']:visible").each(function (i, el) {  
+        let $field = $(this);
+        if ($field.hasClass('custom-currency-input')) {
+            if (!isValidCurrencyCustomField($field)) {
+                invalidFieldsFound = true;
+            }
+        } else {
+            let fieldValidity = document.getElementById(el.id).validity.valid;
+            if (!fieldValidity) {
+                invalidFieldsFound = true;
+            }
         }
+        
+    });
+
+    return invalidFieldsFound;
+}
+
+function formHasInvalidCurrencyCustomFields(formId) {
```

Comment:
these methods should be moved to the GrantApplications/details.js file as intent is to share for all the child tabs not just the AssessmentResults tab (what if we don't load the assessment results tab?)

Reply:
@AndreGAot - done moving the function to GrantApplications/Details.js